packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -126,6 +144,17 @@ const startState = {
  * 2. Keydown (spacebar/enter) -> active state
  * 3. Keyup (spacebar/enter) -> focus state
  *
+ * Warning: The event handlers returned (onClick, onMouseEnter, onMouseLeave,
+ * onMouseDown, onMouseUp, onTouchStart, onTouchEnd, onTouchCancel, onKeyDown,
+ * onKeyUp, onFocus, onBlur, tabIndex) should be passed on to the component
+ * that has the ClickableBehavior. You cannot override these handlers without
+ * potentially breaking the functionality of ClickableBehavior.
+ *
+ * There are internal props triggerOnEnter and triggerOnSpace that can be set
+ * to false if one of those keys shouldn't count as a click on this component.
+ * Be careful about setting those to false -- make certain that the component
+ * shouldn't process that key.
+ *
```

Comment:
It'd be awesome to link to the doc you wrote up in here, as well! I think people would find it to be very useful when trying to understand this problem better.